### PR TITLE
#163602556 persist data in database

### PR DESCRIPTION
**Pull request by Jesse-efe**
#### What does this PR do?
Integrate postgres database to project
#### Description of Task to be completed?

Persist App data in database

Write SQL queries to store and get data from database

Test API endpoints
#### How should this be manually tested?
- Pull this repository and cd into it
- Run npm install to install all dependencies
- Run npm start to start the app
- Use Postman to test the routes listed readme.md

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#163552796 #163602556
#### Screenshots
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by akinwale-habibullah — COMMENTED** [review]
Add a sample.env file in your project root folder, and add the required variables that your project needs to successfully connect to the db.

Another thing I will like to see is a guide in your README file, detailing what I need to configure to get your app running locally.

@Jesse-efe

**Comment by Jesse-efe** [comment]
Ok

**Review by akinwale-habibullah — COMMENTED** [review]
You dont have a sample.env file in your repo as you instructed in your readme. @Jesse-efe

File: source/models/config.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Pool } from 'pg';
+
+require('dotenv').config();
```
Comment (akinwale-habibullah) [inline source/models/config.js]: Why using require along with import ?

File: source/models/tables.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+       id SERIAL,
+       type VARCHAR(50),
+       name VARCHAR(50),
+       PRIMARY KEY (ID)
+    )`;
+  await pool.query(sql);
+  console.log('created');
+};
+
+export default seedDb;
```
Comment (akinwale-habibullah) [inline source/models/tables.js]: Please add some data to your app. and create at least one admin record, which uses an environment variable specified by the user.

File: source/models/config.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+const pool = new Pool({
+  // connectionString: process.env.devConnectionString,
+  connectionString: process.env.testConnectionString,
```
Comment (akinwale-habibullah) [inline source/models/config.js]: You need to have better logic to help automate this, instead of relying on the user to change values here.

**Comment by Jesse-efe** [comment]
Ok thanks, I have implemented the feedback

**Pull request merged**